### PR TITLE
Added summary page and a submenu

### PR DESCRIPTION
**Pull request by 602dhruviii**
# Description
added Colored Backgrounds for Alert States, "for" value should be less than evaluation interval - UI, Popup for Alert Name Input on Logs Page 
Changed table layout and added functionality to searchbar.

## Previous
Screenshot 2024-06-19 235544

## New 
Screenshot 2024-06-20 174734

----------------------->
Screenshot 2024-06-20 174336
Screenshot 2024-06-20 174411
Screenshot 2024-06-20 174608

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: static/summary.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```
Comment (nkunal) [inline static/summary.html:1]: Rename this file to be metricsummary

File: static/js/summarytable.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const metricsPerPage = 28;
+    let currentPage = 1;
+    let metrics = [];
+    let sortedMetrics = [];
+
+    const fetchMetrics = async () => {
+        const response = await fetch('http://playground.siglens.com:5122/metrics-explorer/api/v1/metric_names', {
```
Comment (nkunal) [inline static/js/summarytable.js:8]: this should not point to playground, it was there just for testing, follow how we do for other api calls

File: static/metricsummary.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Summary Table</title>
```
Comment (sonamgupta21) [inline static/metricsummary.html:6]: 1. Put the title as `SigLens`.
2. Look at other HTML files and add similar copyright comments at the top, as well as other gtags and meta tags.
3. Rename `metricssummary.html` to `metric-summary.html` as we use kebab case in other HTML files. Also, rename the JS file to `metric-summary.js`.
4. Do not add a separate CSS file; move everything to the siglens.css file.

File: static/metricsummary.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    <script src="./js/lib/date_fns.min.js"></script>
+    <script src="./js/lib/popper.min.js"></script>
+    <script src="./js/lib/bootstrap.bundle.min.js"></script>
+    <script src="./js/lib/ag-grid-community.min.noStyle.js"></script>
+    <script>
+        var defaultTheme = Cookies.get('theme') || 'light';
+        $('html').attr('data-theme', defaultTheme);
+    </script>
+    {{ .RunCheck1 | safeHTML }}
+    <script src="./js/summarytable.js"></script>
```
Comment (sonamgupta21) [inline static/metricsummary.html:28]: Move this script tag to the end of the HTML file.

**Comment by sonamgupta21** [comment]
image
The screen should not be vertically scrollable, and the pagination should be visible without scrolling. Right now, the user has to scroll down to see the pagination.

File: static/css/siglens.css [hunk 5]
```diff
@@ -5355,6 +5487,151 @@ input.form-control {
...
+  overflow-y: scroll;
+  scrollbar-width: none; 
+  -ms-overflow-style: none; 
+}
+.head{
+  margin-top: 0.5%;
+}
+
+
+.pagination {
```
[the remark this answers is not in the document]
Comment (602dhruviii) [reply static/css/siglens.css:5505]: checktimeopt and checktimebtn are used in metrics summary dateextraction

File: static/css/siglens.css [hunk 6]
```diff
@@ -2084,6 +2110,112 @@ div.dts div.dataTables_scrollBody table {
...
+    width: 300px;
+  }
+  
+  .header {
+    margin-top: 0;
+    margin-bottom: 20px;
+    font-size: 1.5em;
+  }
+  
+  label {
```
Comment (sonamgupta21) [inline static/css/siglens.css:2170]: here , you have given CSS style without and specific id or class ( label, input, button) 
Remove them and it should fix the other screens.

File: et --hard 77da871711782fcc7a47445aaf2f901029d15db6
Comment (AndrewHess) [inline et --hard 77da871711782fcc7a47445aaf2f901029d15db6:1]: Remove this file

**Comment by 602dhruviii** [comment]
closed PR

**Pull request closed without merging**